docs/modules/reference/pages/service-assurance/monitors/SSLCertMonitor.adoc

```diff
@@ -74,9 +74,9 @@ An additional preliminary message and response regular expression pair is availa
 | n/a
 |===
 
-kbd:[{}] _indicates the parameter supports <<service-assurance/monitors/introduction.adoc#ga-service-assurance-monitors-placeholder-substitution-parameters, placeholder substitution>>._
+kbd:[{}] _indicates the parameter supports <<reference:service-assurance/introduction.adoc#ref-service-assurance-monitors-placeholder-substitution-parameters, placeholder substitution>>._
 
-This monitor implements the <<service-assurance/monitors/introduction.adoc#ga-service-assurance-monitors-common-parameters, Common Configuration Parameters>>.
+This monitor implements the <<reference:service-assurance/introduction.adoc#ref-service-assurance-monitors-common-parameters, Common Configuration Parameters>>.
 
```

Comment:
Line 111 has an extra `\` that shouldn't be there.  The curly brackets don't need to be escaped within a source block.